Celeste.Mod.mm/Patches/TextMenu.cs

```diff
@@ -587,6 +594,31 @@ public static void PatchTextMenuOptionColor(ILContext context, CustomAttribute a
             cursor.Next!.Operand = f_UnselectedColor;
         }
 
+        public static void PatchTextMenuUpdate(ILContext context, CustomAttribute attrib) {
+            TypeDefinition t_CoreModule = MonoModRule.Modder.Module.GetType("Celeste.Mod.Core.CoreModule");
+            TypeDefinition t_CoreModuleSettings = MonoModRule.Modder.Module.GetType("Celeste.Mod.Core.CoreModuleSettings");
+
+            // Get the getters manually because properties scare me
+            MethodDefinition m_CoreModule_get_Settings = t_CoreModule.FindMethod("get_Settings");
+            MethodDefinition m_get_AllowTextHighlight = t_CoreModuleSettings.FindMethod("get_AllowTextHighlight");
+
+            ILCursor cursor = new ILCursor(context);
+            cursor.GotoNext(MoveType.Before, instr => instr.MatchLdsfld("Celeste.Settings", "Instance"));
+            
+            // Put the settings module on the stack
+            cursor.Next.OpCode = OpCodes.Call;
+            cursor.Next.Operand = m_CoreModule_get_Settings;
+
+            // Get the AllowTextHighlight value 
+            cursor.Index++;
+            cursor.Next.OpCode = OpCodes.Call;
```

Comment:
```suggestion
            cursor.Next.OpCode = OpCodes.Callvirt;
```

Same here - `AllowTextHighlight` is an instance property.

Reply:
Addressed in commit [a87a04f](https://github.com/EverestAPI/Everest/pull/836/commits/a87a04fdafc78728fdb24bff92cbe487f25b805b)